src/client/state/actions/app.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
We should probably remove this and handle the consequences of linting :smiley: :laughing:

Reply:
Yes, we have a linter configured, would be stupid to not use it, no matter how much headache it sometimes gives you. Point taken. 👍

Reply:
Also, feel free to edit the linter to your liking. We should probably stop using airbnb's restrictive styleguide and come up with our own ESLint rules, which would work for this project.